api/src/main/java/org/apache/iceberg/expressions/BoundReference.java

```diff
@@ -36,15 +36,15 @@
     this.type = struct.fields().get(pos).type();
   }
 
-  private int find(int fieldId, Types.StructType struct) {
+  private int find(int id, Types.StructType struct) {
```

Comment:
Is this something triggered by the plugin?

Reply:
Parameters cannot have the same name as fields.